siftseek/endpoints/seeker/apply/seeker_apply_put.py

```diff
@@ -0,0 +1,52 @@
+from flask import Response, jsonify, request
+from marshmallow import ValidationError
+
+from siftseek.models.db import db
+from siftseek.models.job_application import JobApplication
+from siftseek.endpoints.seeker import seeker
+from siftseek.endpoints.helpers.query_helpers import get_model_by_pk_or_404
+from siftseek.schemas.job_application_schema import application_schema
+
+
+@seeker.put("/apply/<int:seeker_id>")
+def update_job_application(seeker_id: int) -> tuple[Response, int]:
```

Comment:
i'm not sure if you'll need this route. I feel like once you apply for a job, you can't edit the job application info after you submit it. Or at least that's how it is on LinkedIn when i apply for jobs. But up to you ultimately.

Reply:
Good point!